si_ps_conv_fastcorrsApp/Db/kicks.template

```diff
@@ -0,0 +1,219 @@
+    field(DTYP, "Python Device")
+    field(OUT, "@strn2curr $(SEC) $(DIP) $(ORI) Current-SP")
+}
+
+record(ai, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(SCAN, "Passive")
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB PP")
+}
+
+record(ai, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(INP, "@curr2strn $(SEC) $(DIP) $(ORI) Current-SP")
+    field(SCAN, "Passive")
+    field(FLNK, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB")
+}
+
+record(ao, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DOL, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB")
+    field(OMSL, "closed_loop")
+    field(OUT, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP PP")
+}
+
+record(ai, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-DRVL") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(INP, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP.DRVL CP")
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVL PP")
+}
+
+record(ai, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVL") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(SCAN, "5 second")
+    field(INP, "@curr2strn $(SEC) $(DIP) $(ORI) Current-DRVL")
+    field(FLNK, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVL")
+}
+
+record(ao, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVL") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DOL, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVL")
+    field(OMSL, "closed_loop")
+    field(OUT, "SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP.DRVL")
+}
+
+record(ai, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-DRVH") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(INP, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP.DRVH CP")
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH PP")
+}
+
+record(ai, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(SCAN, "5 second")
+    field(INP, "@curr2strn $(SEC) $(DIP) $(ORI) Current-DRVH")
+    field(FLNK, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH")
+}
+
+record(ao, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DOL, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-DRVH")
+    field(OMSL, "closed_loop")
+    field(OUT, "SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP.DRVH")
+}
+
+record(sseq, "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-RB") {
+    field(PINI, "YES")
+    field(DOL1, "SI-$(SEC)$(DIP):PS-$(ORI):Current-RB CP")
+    field(LNK1, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-RB PP")
+}
+
+record(bi, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):CurrentRBCnct-Mon") {
+    field(PINI, "YES")
+    field(SCAN, "1 second")
+    field(INP , "$(TEST)proxy-SI-$(SEC)$(DIP):PS-$(ORI):Current-RB.DOL1V")
+    field(ZNAM, "Ext PV Not Connected")
+    field(ZSV , "MAJOR")
+    field(ONAM, "Ext PV OK")
+}
```

Comment:
Now that we understand build-time generation of dbs better, why not have a template that works for all of `Current-RB/Mon`, `CurrentRef`, `FOFBAcc`? That way you need to specify the DB logic only once, and we can instantiate it multiple times